app/server/src/routes/course/course.route.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const CourseController = require("./course.controller");
+const { validate } = require("./course.validate");
+const { handleValidation } = require("../../services/validate");
+
+const courseRouter = express.Router();
+
+courseRouter.post(
+  "/createCourse",
+  // validate('createCourse'),
+  // handleValidation,
+  CourseController.createCourse
+);
+
+courseRouter.get(
+  "/getCourses",
+  // validate('getCourses'),
+  // handleValidation,
+  CourseController.getCourses
+);
+
+courseRouter.get(
+  "/getEnrolledCourses",
+  // validate('getEnrolledCourses'),
+  // handleValidation,
+  CourseController.getEnrolledCourses
+);
+
+courseRouter.post(
+  "/createEnrollment",
+  // validate('createEnrollment'),
+  // handleValidation,
+  CourseController.createEnrollment
+);
+
+courseRouter.get(
+  "/getCourseDetail",
```

Comment:
instead of this, lets just leave it as `/:id`, we dont need to write this long word, when they send a get request to courseRouter, we already know it is to get the course with `id`

Reply:
done!